homeassistant/components/camera/xiaomi.py

```diff
@@ -98,29 +101,34 @@ def get_latest_video_url(self):
             _LOGGER.warning("There don't appear to be any uploaded videos")
             return False
 
-        first_dir = dirs[-1]
-        try:
-            ftp.cwd(first_dir)
-        except error_perm as exc:
-            _LOGGER.error('Unable to find path: %s', first_dir)
-            _LOGGER.debug(exc)
-            return False
+        if self._model == 'xiaofang':
```

Comment:
Define constants at the beginning of the file for all the fixed strings

Reply:
you mean to store 'xiaofang' and 'yi' as constants?